cli/src/test/integration-tests/install-intellij.sh

```diff
@@ -0,0 +1,2 @@
+echo "Running install intellij integration test"
+ide -d install intellij
```

Comment:
Here we should assert that the software folder actually contains intellij by checking some marker files.
Feel free to remove tests (MVP, KISS) from this PR but when I do a review, I keep on raising concerns and expect that a test we have as a starting point is a good example and blueprint. Otherwise we will have the "broken window" effect and other developers will see this and think, this is a good test and will follow to do the same pattern.

Reply:
I've added a new `assertThat 'file' exists` and checked if the version file and the binary of `intellij` exists.